app/components/primer/beta/clipboard_copy.ts

```diff
@@ -37,12 +37,26 @@ document.addEventListener('clipboard-copy', ({target}) => {
   if (!target.hasAttribute('data-view-component')) return
 
   const currentTimeout = clipboardCopyElementTimers.get(target)
+  const clipboardCopyLiveRegion = target.parentNode?.querySelector<HTMLElement>('[data-clipboard-copy-feedback]')
```

Comment:
Can we use a Catalyst target for this?

Reply:
Sure, good call! Do I have to convert the rest of this file to a Catalyst component too? Or is there a way to just use the target without the rest of the component being a Catalyst component?

Reply:
Oh I think the whole thing would need to be a Catalyst component. If that's too much work, then feel free to disregard 😄